src/common/utils/validator.js

```diff
@@ -91,16 +91,16 @@ function valuetrue(value) {
 }
 
 function daysInMonth(m, y) {
-  switch (m - 1) {
-    case 1:
-      return (y % 4 === 0 && y % 100) || y % 400 === 0 ? 29 : 28;
-    case 8:
-    case 3:
-    case 5:
-    case 10:
-      return 30;
+  switch (m) {
+    case '2':
+      return (y % 4 == 0 && y % 100 != 0) || (y % 100 == 0 && y % 400 == 0) ? '29' : '28';
```

Comment:
Question: is there a good reason to use this code over using the native javascript e.g. `new Date(y,m + 1, 0).getDate()` (https://stackoverflow.com/questions/1184334/get-number-days-in-a-specified-month-using-javascript) or an npm library e.g. [luxon](https://www.npmjs.com/package/luxon?activeTab=dependencies), etc, etc?
- Dates and time can be notorious complicated.
- Could contain a bug from edge case.

Though
- Not downloading a library or using edge case behaviour is nice.

Reply:
Has been done, thanks for doing so.